modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -292,16 +295,15 @@ class ONNXWithRemap : public ONNXtest {
 public:
     // You can specify any size of the outputs, since we don't know infer result
     // Tests validate a range with results and don't compare empty space
-    virtual void validate() override {
+    void validate() {
         GAPI_Assert(!out_gapi.empty() && !out_onnx.empty());
         const auto size = std::min(out_onnx.size(), out_gapi.size());
         for (size_t i = 0; i < size; ++i) {
             float* op = out_onnx.at(i).ptr<float>();
             float* gp = out_gapi.at(i).ptr<float>();
             const auto out_size = std::min(out_onnx.at(i).total(), out_gapi.at(i).total());
             for (size_t d_idx = 0; d_idx < out_size; ++d_idx) {
-                // If both outputs used remapping, they have an end-of-data mark
-                if ((gp[d_idx] == -1.f) && (op[d_idx] == -1.f)) {
+                if (gp[d_idx] == -1.f) {
```

Comment:
May be `||`?
Is `gp` always smaller?

Reply:
It is "end data mark" for gapi output.

Reply:
As I understand correctly, this way of validation can allow the following: the output of G-API is greater than ONNX output.
But in that case the objects after `out_size` would not be validated...
Is it correct?

Reply:
G-Api output, that greater than ONNX output contains garbage. We check correct data with size from ONNX or until end of data mark. In remap we can choose any size for gapi output.